src/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics.cpp

```diff
@@ -58,7 +58,7 @@ void Dynamics::DynamicalEquations(
     const Instant& anInstant
 )
 {
-    for (Index i = 0; i < dxdt.size(); ++i)
+    for (Index i = 0; i < (Size)dxdt.size(); ++i)
```

Comment:
```suggestion
    for (Size i = 0; i < (Size)dxdt.size(); ++i)
```

I know they're the same under the hood, but this looks a bit nicer IMO

Reply:
Agreed!

Reply:
Agreed as well, Thanks for thinking this over!